django/contrib/postgres/fields/array.py

```diff
@@ -88,6 +88,16 @@ def to_python(self, value):
             value = [self.base_field.to_python(val) for val in vals]
         return value
 
+    def from_db_value(self, value, expression, connection, context):
+        if value is None:
+            return value
+        return [
+            self.base_field.from_db_value(val, expression, connection, context)
+            if hasattr(self.base_field, 'from_db_value') else
+            self.base_field.to_python(val)
```

Comment:
I'm not sure in what cases `to_python` and `from_db_value` are called. So, I added the fallback just in case. If you think it's wired then it can be probably wrong. Shall I submit a new version without that fallback?